quick_monolith_install/cchq-install.sh

```diff
@@ -65,9 +65,6 @@ printf "\nStep 3: Initializing environments directory for "
 printf "storing your CommCareHQ instance's configuration \n"
 printf "#################################################"
 printf "\n"
-# VENV should have been set by init.sh
```

Comment:
I like that it's red here. Setting a default like this (I know I did it) is...less desirable. Thanks for putting in time to investigate this deeper! I totally missed the pipe operator so I couldn't figure out why the  variable is not set in the correct scope.